lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
+  <%%= f.error_notification %>
+  <%%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
```

Comment:
- [ ] This if statement should be put in a helper method instead of having it straight in the view